docs/docs/workflows/create.md

```diff
@@ -29,6 +32,51 @@ For details on the flags, consult the command help via `constellation create -h`
 
 *create* stores your cluster's state into a [`terraform.tfstate`](../architecture/orchestration.md#cluster-creation-process) file in your workspace.
 
+</tabItem>
+<tabItem value="terraform" label="Terraform">
+
+:::caution
+
+There is no extensive input validation with Terraform. If unsure, please use the Constellation CLI instead.
+
+:::
+
+Constellation also supports managing the infrastructure via Terraform. This allows for an easier GitOps integration as well as meeting regulatory requirements.
+Since the Constellation CLI also uses Terraform under the hood, the same Terraform files can be reused.
```

Comment:
```suggestion
Constellation supports managing the infrastructure via Terraform. This allows for an easier GitOps integration as well as meeting regulatory requirements.
Since the Constellation CLI also uses Terraform under the hood, you can reuse the same Terraform files.
```